CHANGELOG/CHANGELOG-3.7.md

```diff
@@ -10,6 +10,10 @@ Previous change logs can be found at [CHANGELOG-3.6](https://github.com/etcd-io/
 
 - [Removed all deprecated experimental flags](https://github.com/etcd-io/etcd/pull/19959)
 
+### etcd server
+
+- [Update go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc to v0.61.0](https://github.com/etcd-io/etcd/pull/20017)
```

Comment:
I think it'd better to explicitly clarify that we replaced the deprecated `UnaryServerInterceptor` and `StreamServerInterceptor` with `NewServerHandler`. Something like below,
```suggestion
- [Update go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc to v0.61.0 and replaced the deprecated `UnaryServerInterceptor` and `StreamServerInterceptor` with `NewServerHandler`](https://github.com/etcd-io/etcd/pull/20017)
```